Library/Formula/openssl.rb

```diff
@@ -17,6 +18,9 @@ class Openssl < Formula
 
   depends_on "makedepend" => :build
 
+  keg_only :provided_by_osx,
```

Comment:
@mistydemeo & @tdsmith seemed to raise objection to that in the chat - Will let them comment though.